scripts/test-in-lxd.sh

```diff
@@ -2,34 +2,40 @@
 set -eux
 IMAGE=$1
 SCRIPT=$2
+TESTER=${IMAGE##*:}
```

Comment:
This is looking great.
However, I think I want to rename the test image so it is less likely to conflict with existing containers.
What do you think about this?
```suggestion
TESTER=subiquity-${IMAGE##*:}
```

Reply:
Sure it's safer than `TESTER=tester-${IMAGE##*:}`. I'll push the change in a bit.